unicodetools/data/ucd/dev/UnicodeData.txt

```diff
@@ -409,7 +409,7 @@
 0198;LATIN CAPITAL LETTER K WITH HOOK;Lu;0;L;;;;;N;LATIN CAPITAL LETTER K HOOK;;;0199;
 0199;LATIN SMALL LETTER K WITH HOOK;Ll;0;L;;;;;N;LATIN SMALL LETTER K HOOK;;0198;;0198
 019A;LATIN SMALL LETTER L WITH BAR;Ll;0;L;;;;;N;LATIN SMALL LETTER BARRED L;;023D;;023D
-019B;LATIN SMALL LETTER LAMBDA WITH STROKE;Ll;0;L;;;;;N;LATIN SMALL LETTER BARRED LAMBDA;;;;
+019B;LATIN SMALL LETTER LAMBDA WITH STROKE;Ll;0;L;;;;;N;;;A7DC;;A7DC
```

Comment:
Looks like we lost the Unicode_1_Name!

There should be an invariant test for that.

Reply:
I think that's a mistake/oversight in the proposal. We should not lose the Unicode_1_Name.

Reply:
Indeed.